File: cpp/Two.cpp
#include <string>
#include <iostream>

int main() {
  std::cout << "What is the input string?";
  std::string inpString;
  std::cin >> inpString;
  std::cout << inpString << " has " << inpString.length() << " characters.";
  return 0;
}

File: rust/examples/two.rs
fn main() {
    println!("What is the input string?");
    let mut name = String::new();

    std::io::stdin().read_line(&mut name)
        .expect("Failed to read line");
    let count = name.trim_end().chars().count();
     
    println!("{} has {} characters.", name.trim_end(), count);


}
